mir_eval/sonify.py

```diff
@@ -131,12 +131,18 @@ def _fast_synthesize(frequency):
         # Get a waveform of length samples at this frequency
         wave = _fast_synthesize(frequency)
         # Scale each time interval by the piano roll magnitude
-        for m, (start, end) in enumerate((times * fs).astype(int)):
+        for m, (start, end) in enumerate((times[:-1, :] * fs).astype(int)):
```

Comment:
This means that the final column in `gram` will never be accessed, I think.  You need to special-case this after the for loop.